java/ql/src/Security/CWE/CWE-338/types.qll

```diff
@@ -0,0 +1,52 @@
+import java
+
+class UUIDCreationExp extends ClassInstanceExpr {
+  UUIDCreationExp() { this.getConstructedType() instanceof TypeUUID }
+}
+
+class ApacheRandomStringUtilsType extends RefType {
+  ApacheRandomStringUtilsType() {
+    this.hasQualifiedName("org.apache.commons.lang", "RandomStringUtils") or
+    this.hasQualifiedName("org.apache.commons.lang3", "RandomStringUtils")
+  }
```

Comment:
Have you tried non-string variants (eg: org.apache.commons.lang3.RandomUtils)? do they generate a high rate of FPs?

Reply:
Can you elaborate on what you mean?

Reply:
You are accounting for `org.apache.commons.lang3.RandomStringUtils`, but not for `org.apache.commons.lang3.RandomUtils`. Did you miss that class or did you discard it because it was generating too many false positives?

Reply:
Huh, nope, I just didn't know it existed, thanks!